torchx/lib/torchx.ex

```diff
@@ -17,4 +19,71 @@ defmodule Torchx do
   """
   def device_count(:cuda), do: NIF.cuda_device_count()
   def device_count(_), do: raise("Only CUDA devices can be counted for now.")
+
+  # LibTorch API bindings
+
+  ## Creation
+
+  def arange(from, to, step \\ 1, opts \\ []) do
+    type = opts[:type] || Nx.Type.infer([from, to])
+    device = opts[:device] || :cpu
+
+    NIF.arange(from, to, step, torch_type(type), torch_device(device))
+    |> wrap_with_device()
```

Comment:
In my mind we would do this:

```suggestion
    device = opts[:device] || :cpu

    NIF.arange(from, to, step, torch_type(type), torch_device(device))
    |> wrap_with_device(device)
```

We would always pass the device in.